app/jobs/sqs_worker.rb

```diff
@@ -0,0 +1,22 @@
+class SqsWorker
+  include Shoryuken::Worker
+
+  # TODO: set real queue name
+  # TODO: set auto_delete: true after testing
+  shoryuken_options queue: 'TODO-add-real-queue', body_parser: :json, auto_delete: false
```

Comment:
@qrush:
> Do we actually need Shoryuken here? What's the advantage to using this when we have Delayed::Job?

We need a tool that uses AWS SQS as the queue. See [this comment & discussion on rubygems-infrastructure](https://github.com/rubygems/rubygems-infrastructure/issues/35#issuecomment-160700892).